pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -553,6 +547,21 @@ private void createPersistentSubscriptions() {
         checkReplicatedSubscriptionControllerState();
     }
 
+    private CompletableFuture<Void> removeOrphanReplicationCursors() {
+        List<CompletableFuture<Void>> futures = new ArrayList<>();
+        List<String> replicationClusters = topicPolicies.getReplicationClusters().get();
+        for (ManagedCursor cursor : ledger.getCursors()) {
+            if (cursor.getName().startsWith(replicatorPrefix)) {
+                String remoteCluster = PersistentReplicator.getRemoteCluster(cursor.getName());
+                if (!replicationClusters.contains(remoteCluster)) {
+                    log.warn("Remove the replicator because the cluster '{}' does not exist", remoteCluster);
```

Comment:
Please feel free to remove this warning. The removeReplicator will print an info log when a replicator will be removed.

Reply:
I want to leave this log there because this log helped me to find this issue 😂

Reply:
Please make the log message unique

Reply:
Yes, it is already unique. I added another key word `orphan`